src/components/main-page/Location/LocationMap.tsx

```diff
@@ -0,0 +1,34 @@
+import { MapContainer, Marker, TileLayer } from "react-leaflet";
+import "leaflet/dist/leaflet.css";
+import styles from "./LocationMap.module.scss";
+import { LatLngExpression } from "leaflet";
+import markerIconPng from "leaflet/dist/images/marker-icon.png";
+import { Icon } from "leaflet";
```

Comment:
`leaflet` is imported twice 😉 

```suggestion
import { Icon, LatLngExpression } from "leaflet";
import markerIconPng from "leaflet/dist/images/marker-icon.png";
```